torch/distributed/run.py

```diff
@@ -728,6 +728,8 @@ def config_from_args(args) -> Tuple[LaunchConfig, Union[Callable, str], List[str
 
     log_line_prefix_template = os.getenv("TORCHELASTIC_LOG_LINE_PREFIX_TEMPLATE")
 
+    log_for_fb_tupperware = os.getenv("TORCHELASTIC_LOG_FOR_FB_TUPPERWARE", "") == "1"
```

Comment:
is the proposal to use an ENV bc we want to not require any api changes on the internal customers? otherwise a cmdline arg or even a runtime API call might be sufficient.

(we could have a fb/utils_internal function called get_logdir() that  returns one path in fbcode and on oss it creates the the subdir and returns that path instead).  

the feature could be gated by env for rollout purposes and then the env could go away later if we leave it on all the time internally.

Reply:
I was originally planning to do it as a command line flag, but @kurman suggested doing an environment variable because it isn't really intended (as currently formulated) to be public API. I honestly do not care, y'all figure it out and I'll implement it however you want it.

Reply:
no strong opinion other than trying to not have too many configs. if we can reuse existing arguments for specifying logdir and add special behavior when we specify meta logdir on meta machines that might be a way.  but ENV is fine with me too.

Reply:
What do you think about the "log name template" string concept? Default is `{rdzv_id}/attempt_{attempt}/{name}.{ext}` but then you can replace it with `dedicated_log_{rdzv_id}_attempt_{attempt}_{name}.{ext}`

Reply:
well, I wonder if extra flexibility is a real need here, or if trying to force people onto the standard (with the admission that the old format has legacy and should still be supported for some time) is better overall.  Also if we are in the mood to be flexible, is that set of 4 placeholders going to be enough for anyone who wants flexibility or are they just going to have to change the code to add different placeholders anyway?

Reply:
I am literally trying to social engineer people into being OK with  some FB specific logic, I don't care how what we end up with, just need people to be comfortable with it LOL